tensorbay/opendataset/Cityscapes/loader.py

```diff
@@ -14,8 +14,76 @@
 from tensorbay.dataset import Data, Dataset
 from tensorbay.label import InstanceMask, LabeledPolygon, SemanticMask
 
-DATASET_NAME = "CityscapesGTFine"
-_SEGMENT_NAMES = {"test", "train", "val"}
+DATASET_NAME_GTCOARSE = "CityscapesGTCoarse"
+DATASET_NAME_GTFINE = "CityscapesGTFine"
+_SEGMENT_NAMES_GTCoarse = ("train", "train_extra", "val")
```

Comment:
The variable name `_SEGMENT_NAMES_GTCoarse` is a mix of UPPER_CASE and camelCase